app/models/repo.rb

```diff
@@ -70,7 +70,9 @@ def with_travis_service
   end
 
   def enabled_for?(checker)
-    Array(Settings.public_send(checker).enabled_repos).include?(name)
+    Array(Settings.public_send(checker).enabled_repos).each_with_object([]) { |value, values|
+      values << (value.kind_of?(Array) ? value.first.to_s : value)
+    }.include?(name)
```

Comment:
No, the code smell is that 9/10 times when people use each_with_object with an Array, it can just be changed into a regular method call on Array, like .collect.